uint8 leaf_id
---

uint8 exception_code
uint8[] data